ImageNet/inference/Intel_resnet50_c59xlarge_intelcaffe.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "v1.0",
+    "author": "Intel(R) Corporation",
+    "authorEmail": "daisy.deng@intel.com",
+    "framework": "Intel(R) Optimized Caffe",
+    "codeURL": "https://github.com/intel/caffe/tree/DAWN_Bench",
```

Comment:
@daisyden Can you either change this to a specific commit hash or include the commit hash in `misc`?

Reply:
@codyaustun I will update the README file to provide all the compile command. The error you met should caused by the lmdb dataset. In my test we used raw lmdb without resizing and used "caffe test" command for real inference. Since reading data from memory is ok, I will use command "caffe time --forward_only" on dummy data. The latency indicator will be better. I will let you know when the result is updated to github.

Reply:
@codyaustun if my data is updated, do I need to submit a new PR? Except to use caffe time, I will also adopt a new feature of Intel Caffe to make the performance even better.

Reply:
@codyaustun another question is if we run Caffe training on AWS with up to 128 nodes, is it ok for training time competition? And you will help to reproduce the data on 128 nodes, right?

Reply:
You can either update this pull request or create another one. Either is fine.

Training on multiple nodes is fine but try to make your command and code general enough that we can reproduce with a smaller configuration.

Reply:
@daisyden please don't use dummy data for the inference time. If we need to validate accuracy, your code must run on real data. Instead please just include instructions on how to store the data for your command.